tests/rptest/services/compatibility/kafka_streams_examples.py

```diff
@@ -171,3 +171,75 @@ def __init__(self, redpanda, is_driver):
 
         super(KafkaStreamsMapFunc, self).__init__(redpanda, is_driver,
                                                   jar_args)
+
+
+class KafkaStreamsInteractiveQueries(KafkaStreams):
```

Comment:
If we split `KafkaStreamsInteractiveQueries` into `KafkaStreamsInteractiveQueriesDriver` and `KafkaStreamsInteractiveQueriesExample` then we don't need to introduce `is_driver` flag and additional branching.